core/server/src/test/java/alluxio/master/block/BlockMasterPrivateAccess.java

```diff
@@ -30,7 +30,7 @@
    */
   public static boolean isWorkerRegistered(BlockMaster master, long workerId) {
     IndexedSet<MasterWorkerInfo> workers = Whitebox.getInternalState(master, "mWorkers");
-    IndexDefinition<MasterWorkerInfo> idIndex = Whitebox.getInternalState(master, "mIdIndex");
+    IndexDefinition<MasterWorkerInfo> idIndex = master.ID_INDEX;
```

Comment:
can we still use whitebox to get the static variable? so we could make it still private?